autoscheduler/frontend/src/types/Contributors.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Represents a Contributor to the project. Only used in our About section where we show
+ * who all works on the project.
+ */
+export interface Contributor {
+  name: string;
+  position: string;
+  githubURL: string;
+  linkedInURL: string | null;
```

Comment:
```suggestion
  linkedInURL?: string;
```
Making this optional makes it so you don't have to specify the URL as null if someone doesn't have one